api/customAttributes.js

```diff
@@ -93,7 +93,22 @@ module.exports = function(app) {
       retObj.stem = id.length === 6 ? stems.includes(id) ? "Stem Major" : false : "Contains Stem Majors"
     }
     else if (dimension === "University") {
+      const loadJSON = require("../utils/loadJSON");
+      const universitySimilar = loadJSON("/static/data/similar_universities_with_dist.json");
```

Comment:
Can you move these imports to the top of the file, outside of the API endpoints? I worry that loading this JSON file inside each instance of this API endpoint could lead to speed issues and/or memory bloat.